apps/mobile-alphabet/android/settings.gradle

```diff
@@ -0,0 +1,7 @@
+rootProject.name = 'MobileAlphabet'
```

Comment:
Be careful around
- whether this needs to line up with other string properties on other config files
- whether this is user-facing (and would require a space)